### PR TITLE
Home Screen Quick-Filter and UI Changes

### PR DESCRIPTION
**Pull request by BryanCaldeira**
Cards:
[155-header-fix-home-screen](https://trello.com/c/2A9QDz0y/155-header-fix-home-screen)
[134-home-load-hazards-around-current-location](https://trello.com/c/1N2slyc6/134-home-load-hazards-around-current-location)
[135-ui-update-home-screen](https://trello.com/c/4ABYXBRV/135-ui-update-home-screen)

### CODE REVIEW

File: assets/css/quick-filter.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+.quick-filter .quick-filter__icon {
+  background-color: var(--neutral-500);
+  width: 1.25rem;
+  height: 1.25rem;
+}
+
+.quick-filter.selected {
+  border-radius: 0.5rem;
+  background: var(--primary-900, #0f1815) !important;
```
Comment (whamester) [inline assets/css/quick-filter.css:29]: This is not needed the hover takes priority, we need different styles for .selected:hover, and we need to ask Design for those styles.
In the meantime, please remove the !important

Comment (BryanCaldeira) [reply assets/css/quick-filter.css:29]: Hey, Without the `!important` property the selected styles aren't getting applied immediately after click event. You can check this in a mobile phone and notice the difference without `!important`

**Comment by whamester** [comment]
- View hazard cards

Current state:
Screenshot 2023-11-05 at 1 37 54 PM

Figma:
Screenshot 2023-11-05 at 1 38 52 PM

- Fix the width of the cards
- Handle the content overflow to allow wrapping
- The scroll of the cards should touch the right side of the screen

**Comment by whamester** [comment]
- Filters and Search

Current state:
https://github.com/whamester/WMDD-4885-Project/assets/57273227/634a0b6f-a9f5-44ee-970b-a37510e35860

https://github.com/whamester/WMDD-4885-Project/assets/57273227/3b247a43-8ab3-40ec-968b-700038a9ed4f


Prototype:

https://github.com/whamester/WMDD-4885-Project/assets/57273227/68d402c5-f201-44c7-86f7-b7bf73433b48

https://github.com/whamester/WMDD-4885-Project/assets/57273227/2a435b27-d9d8-4f47-997f-321ed106404a


-  Quick filters: these should only display the markers on the map when applied
-  Select Location: 
     - This should display the cards and the back button should clear the search input
     - For the autocomplete suggestions allow the container to show 5 cards.

Current State:
Screenshot 2023-11-05 at 1 44 04 PM
Expected:
Screenshot 2023-11-05 at 1 45 04 PM

**Comment by whamester** [comment]
If I search for a location and then clear the input, I can see the top border of the suggestions container. See if you can prevent that.

Screenshot 2023-11-05 at 1 48 30 PM

**Comment by whamester** [comment]
For the geocode restrict the search to only look for locations with
Screenshot 2023-11-05 at 1 51 16 PM
in Canada

**Comment by whamester** [comment]
Screenshot 2023-11-05 at 2 00 25 PM

**Pull request merged**